connectors/citrus-openapi/pom.xml

```diff
@@ -48,7 +48,32 @@
     <dependency>
       <groupId>com.atlassian.oai</groupId>
       <artifactId>swagger-request-validator-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.sun.mail</groupId>
+          <artifactId>mailapi</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
+
+    <!--
+        force com.sun.mail:mailapi to v2 major.
+        citrusframework/citrus-simulator won't build otherwise.
+    -->
+    <dependency>
+      <groupId>com.sun.mail</groupId>
+      <artifactId>mailapi</artifactId>
+      <version>2.0.2</version>
```

Comment:
can we please manage the versions in parent pom in the dependencyManagement section

Reply:
done. see 29ff88e0f1a045a70ed779db6cb0a42647802b8b.